server/graphql/mutations.js

```diff
@@ -320,6 +320,7 @@ const mutations = {
       CollectiveId: { type: new GraphQLNonNull(GraphQLInt) },
       HostCollectiveId: { type: new GraphQLNonNull(GraphQLInt) },
       description: { type: GraphQLString },
+      PaymentMethodId: { type: GraphQLInt },
```

Comment:
No need for this if we decide that we don't allow to increase the balance of a prepaid card. 
I think this would simplify things.